quickstart.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import print_function
```

Comment:
nit: this is not needed if we're using python3

Reply:
@j2kun 
Sure, I can delete that.   Also, the csv file seems to be formatted in Sheets properly.  Maybe you can check on your end before I merge?